tests/unit/utils/skos/test_codelist.py

```diff
@@ -58,8 +60,12 @@ def test_get_codelist_col_title_by_property_url_notation():
     result_code_list_cols = select_codelist_cols_by_dataset_url(
         csvw_metadata_rdf_graph, dataset_url
     )
-    notation_col_name = get_codelist_col_title_by_property_url(
-        result_code_list_cols.columns, CodelistPropertyUrl.SkosNotation
+    result_code_list_primary_key = select_codelist_primarykey_by_dataset_url(
```

Comment:
Do we have any new tests which explicitly test that this works with the new csvcubed outputs?

Reply:
The new tests is added to behave tests.